doc/sphinx-guides/source/developers/coding-style.rst

```diff
@@ -93,6 +93,20 @@ If you just downloaded Netbeans and are using the out-of-the-box settings, you s
 
 If you know of a way to easily share Netbeans configuration across a team, please get in touch.
 
+Bash
+----
+
+Generally, Google's Shell Style Guide at https://google.github.io/styleguide/shell.xml seems to have good advice.
+
+Formatting Code
+~~~~~~~~~~~~~~~
+
+Tabs vs. Spaces
+^^^^^^^^^^^^^^^
+
+Don't use tabs. Use 2 spaces.
```

Comment:
This is not very consistent throughout the codebase.

Are you guys aware of [Checkstyle Maven plugin](https://maven.apache.org/plugins/maven-checkstyle-plugin/)?
[Example config used Peerpub](https://github.com/peerpub/peerpub/blob/development/checkstyle.xml)

Reply:
@poikilotherm I know! Help! Actually, can you please create an issue just about CheckStyle? To help us with consistent formatting like tabs vs. spaces? We try to work in small chunks and in the future you'd be welcome to create issues for FindBugs and other tools too. If you want, you can write the issue title in the form of a user story with something like "As a developer, I'd like some tooling to report on if the code I'm writing meets the coding style of the project such as tabs vs. spaces." Whatever wording makes sense to you. Then in the issue we can discuss CheckStyle vs. other tools. Again, the point is to have the smallest chunk possible. Maybe just the tabs vs. spaces thing. I hope this makes sense!

Reply:
@poikilotherm thanks for opening #5075 about code style.

Reply:
Cross-referencing: have a look at #5094 :smile: